src/iso.js

```diff
@@ -35,17 +39,18 @@ ${data}
     )
   }
 
-  static render(html, state = {}, meta = {}) {
-    return new Iso().add(html, state, meta).render()
+  static render(html, state = {}, meta = {}, config = { markupClassName: '___iso-html___', markupElement: 'div', dataClassName: '___iso-state___', dataElement: 'div'}) {
+    return new Iso(config).add(html, state, meta).render()
   }
 
-  static bootstrap(onNode) {
+  static bootstrap(onNode, markupClassName = '___iso-html___', dataClassName = '___iso-state___') {
```

Comment:
yeah this is just sugar

otherwise you can create an Iso instance yourself, there's already a bootstrap method on the class.